rest.php

```diff
@@ -98,16 +99,33 @@
     throw new sharing_cart_exception('invalidoperation');
 
 } catch (Exception $ex) {
-    header('HTTP/1.1 400 Bad Request');
+
+	header('HTTP/1.1 400 Bad Request');
+
     $json = array(
-            'message' => $ex->getMessage(),
+        'message' => $ex->getMessage(),
     );
+
     if (!empty($CFG->debug) && $CFG->debug >= DEBUG_DEVELOPER) {
         $json += array(
-                'file' => substr($ex->getFile(), strlen($CFG->dirroot)),
-                'line' => $ex->getLine(),
-                'trace' => format_backtrace($ex->getTrace(), true),
+            'file' => substr($ex->getFile(), strlen($CFG->dirroot)),
+            'line' => $ex->getLine(),
+            'trace' => format_backtrace($ex->getTrace(), true),
         );
     }
+
+	/**
+	 * If logger exists - we log some stuff
+	 */
+    if(class_exists(mysql_logger::class)){
+    	try{
+		    $logger = new mysql_logger();
+		    $logger->log($ex->getMessage(), $ex);
+	    }
+	    catch(\Exception $e){
+//    		echo '<pre>'; var_dump($e); echo '</pre>'; die();
```

Comment:
Skal det ikke bare væk?

Reply:
Jo, det var kun for debugging skyld, men jeg kan godt lige fjerne det.